eiffel-gerrit-plugin-script

```diff
@@ -0,0 +1,138 @@
+#! /bin/bash
+
+GERRIT_API_HOST=""
+STATUS=0
+
+function invalid_arguments {
+  echo "missing command(s) start | stop | test | only_test | restart | build | rebuild"
+  echo "you may combine any commands in the order you wish to execute them"
+  echo " "
+  echo " start     :: start the docker containers"
+  echo " stop      :: stop the docker containers"
+  echo " test      :: build, start, test and stop docker containers"
+  echo " only_test :: executes the test, docker containers must be running"
+  echo " restart   :: restarts the docker containers"
+  echo " build     :: will build the docker images to be used"
+  echo " rebuild   :: see --> build"
+  echo " "
+  echo "Example:"
+  echo "./eiffel-gerrit-plugin-script test"
+  echo "      --> Will compile the source code to an executable jar and "
+  echo "          Build the docker file and start the docker-compose"
+  echo "          Execute the tests and then stop the running services"
+
+  exit;
+}
+
+function do_stop {
+  echo "########################################"
+  echo "###### STOPING DOCKER SERVICES #########"
+  echo "########################################"
+  docker-compose -f target/docker-compose.yml down
+}
+
+function do_rebuild {
+  echo "########################################"
+  echo "######### BUILDING JARS ################"
+  echo "########################################"
+
+  mvn clean install package -DskipTests
+
+  mv target/test-classes/docker-compose.yml target/docker-compose.yml
```

Comment:
Any particular reason to why this is moved into the target directory? In my opinion it adds unnecessary confusion and from a glance I can not immediately understand why we perform this step.

Reply:
I would rather use "cp" and not "mv" atleast? :)

Reply:
Not sure why, but not doing this step will make everything fail in strange way and creates containers with strange names and stuff.